keps/sig-scheduling/624-scheduling-framework/README.md

```diff
@@ -683,24 +683,21 @@ interact with external components of Kubernetes. For example, if a plugin needs
 to interact with the API server and Kubelets, end-to-end tests may be needed.
 End-to-end tests are not needed when integration tests can provided adequate coverage.  
 
-# GRADUATION CRITERIA
+## Graduation Criteria
 
-* Alpha
-  * Extension points for `Reserve`, `Unreserve`, and `Prebind` are built.
+* Alpha (1.16)
+  * Extension points for `Reserve`, and `Prebind` are built.
   * Integration tests for these extension points are added.
 
-* Beta
+* Beta (1.17)
   * All the extension points listed in this KEP and their corresponding tests
   are added.
   * Persistent dynamic volume binding logic is converted to a plugin.
 
-* Stable
+* Stable (1.19)
   * Existing 'Predicate' and 'Priority' functions and preemption logic are
   converted to plugins.
   * No major bug in the implementation of the framework is reported in the past
   three months.
 
-# IMPLEMENTATION HISTORY
```

Comment:
Keep this section? At least with 2 bullets indicating when we transform old format design doc to KEP, and the latest changes.

Reply:
the history is long and complicated :) not sure those two bullet points will be useful.